### PR TITLE
Fix "Download File" link in response-body in Internet Explorer

### PR DESCRIPTION
**Pull request by kovyfive**
The error can be seen on this :
http://swashbuckletest.azurewebsites.net/swagger/ui/index#/Image/Image_Put
When opened with Internet explorer, clicking the link does nothing.

### Description

In internet explorer, the "Download File" button is shown as a link, but when clicked nothing happens.
Right clicking the link and selecting Save as... will ignore file name, and only display the blob name.
The file has the proper content though. 

Internet explorer uses window.navigator.msSaveOrOpenBlob to save files.

Branching in this file, will allow proper functionality to the link, while having no breaking changes.

This way, the internet explorer little window in the middle of the bottom of the screen will pop up, with the proper filename.

### Motivation and Context

The download link is not working in IE

Fixes https://github.com/swagger-api/swagger-ui/issues/3364

### How Has This Been Tested?

Built using NPM, added to WebApi project, built,  opened Swagger, clicked on Download File link, and asserted the Save window appeared.

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [x] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by heldersepu** [comment]
IE the best browser to download a real browser

**Comment by kovyfive** [comment]
Our clients uses Internet Explorer sadly, so we need to support them too. 
Can you help me out why the CI failed?

**Comment by heldersepu** [comment]
Looks like travis is acting up:
```
W: The repository 'http://ppa.launchpad.net/couchdb/stable/ubuntu trusty Release' does not have a Release file.
W: Failed to fetch https://packagecloud.io/computology/apt-backport/ubuntu/dists/trusty/InRelease  Failed to connect to packagecloud.io port 443: Connection timed out
W: Failed to fetch http://ppa.launchpad.net/chris-lea/redis-server/ubuntu/dists/trusty/main/binary-amd64/Packages  Could not connect to ppa.launchpad.net:80 (91.189.95.83), connection timed out
W: Failed to fetch http://ppa.launchpad.net/chris-lea/redis-server/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/chris-lea/redis-server/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/git-core/ppa/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/git-core/ppa/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/git-core/ppa/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/openjdk-r/ppa/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/openjdk-r/ppa/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/openjdk-r/ppa/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/pollinate/ppa/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/pollinate/ppa/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/pollinate/ppa/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/webupd8team/java/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/webupd8team/java/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/webupd8team/java/ubuntu/dists/trusty/main/i18n/Translation-en  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/main/binary-amd64/Packages  Unable to connect to ppa.launchpad.net:http:
W: Failed to fetch http://ppa.launchpad.net/couchdb/stable/ubuntu/dists/trusty/main/binary-i386/Packages  Unable to connect to ppa.launchpad.net:http:
W: Some index files failed to download. They have been ignored, or old ones used instead.
```

Something like that happened to me, I just sent a commit to re-trigger the build in a few minutes and it completed sucessfully

**Comment by heldersepu** [comment]
Now you do have a valid error:
>  51:2  error  Mixed spaces and tabs  no-mixed-spaces-and-tabs

**Comment by kovyfive** [comment]
All good :)

**Comment by kovyfive** [comment]
Whopsie, on the close  :D

**Comment by heldersepu** [comment]
Strange, I still see some trailing tabs ....
image
Maybe travis does not care about those?

**Comment by kovyfive** [comment]
There is no check for that it seems. Removed them.

**Comment by heldersepu** [comment]
AwesomO

**Comment by kovyfive** [comment]
Updated!

File: src/core/components/response-body.jsx [hunk 1]
```diff
@@ -47,7 +47,11 @@ export default class ResponseBody extends React.Component {
           }
         }
 
-        bodyEl = <div><a href={ href } download={ download }>{ "Download file" }</a></div>
+        if(window.navigator.msSaveOrOpenBlob) {
```
[the remark this answers is not in the document]
Comment (kovyfive) [reply src/core/components/response-body.jsx]: Done!

**Comment by shockey** [comment]
thanks, @kovyfive!

**Pull request merged**